CONTRIBUTING.md

```diff
@@ -126,6 +126,9 @@ From there you will have to wait on one of the GAP committers to respond to
 the request. This response might be an accept or some
 changes/improvements/alternatives will be suggested.  We do not guarantee
 that all requests will be accepted.
+Feel free to read the section
+[the reviewing process](#the-reviewing-process)
```

Comment:
I would not use a hyperlink here. It makes the document more annoying to read without a Markdown renderer, and IMHO is not that useful with one either.

Reply:
Ok, that's a point I didn't consider.